hw/ip/prim/dv/prim_prince/crypto_dpi_prince/crypto_dpi_prince.core

```diff
@@ -0,0 +1,18 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:dv:crypto_dpi_prince:0.1"
+description: "PRINCE block cipher reference C implementation from Sebastien Riou"
+filesets:
+  files_dv:
+    files:
+      - prince_ref.h: {file_type: cSource, is_include_file: true}
+      - crypto_dpi_prince.c: {file_type: cSource}
```

Comment:
Do you still need the `{file_tyep: cSrouce}` if line 13 already declared it?

Reply:
Probably remove the one below.

Reply:
good point @cindychip - @sriyerg is right, can remove the one below